devel/libtapi/Portfile

```diff
@@ -23,6 +23,11 @@ checksums               rmd160  a99aa279e626d4d638281ef9555d9b5db5f15145 \
                         sha256  66c715baaea6f28e8c079f36189027a52744138c38e83e97aeb399232365138f \
                         size    32205809
 
+# use cmake-bootstrap to minimize dependencies.
+depends_build-replace   path:bin/cmake:cmake port:cmake-bootstrap
```

Comment:
why not use
```
depends_build-replace   path:bin/cmake:cmake path:bin/cmake:cmake-bootstrap
```

there should be no need to install the `cmake-bootstrap` port if someone already hat the `cmake` port installed - correct?

Reply:
@Gcenx you may add one more `if` to enforce `cmake-bootstrap` on PowerPC. I assume that you're building for PowerPC, aren't you?

Reply:
@Gcenx as far as I know it is.

Reply:
@Gcenx my point: right now it it uses `cmake-bootstrap` for everything before `10.13` but not for PowerPC. I think that enforce it for PowerPC good idea.

Enforce to use cmake-bootstrpa globaly is good idea, that makes such build easy to reproduce.

=> LGTM.

Reply:
What's the status here?

Reply:
@pmetzger from my point of view it should be merged.